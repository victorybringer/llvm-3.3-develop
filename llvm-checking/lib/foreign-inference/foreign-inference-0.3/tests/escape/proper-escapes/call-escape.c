extern int * glob;
void g(int * p, int * b) {
  glob = p;
}

void f(int * q, int a) {
  g(q, &a);
}
