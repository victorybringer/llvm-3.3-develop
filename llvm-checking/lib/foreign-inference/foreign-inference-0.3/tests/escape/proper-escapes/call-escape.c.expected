fromList [ ("@g", fromList [(DirectEscape, "%p")] )
         , ("@f", fromList [(DirectEscape, "%q")] )
         ]
